x/stakeibc/keeper/keeper_test.go

```diff
@@ -67,16 +68,50 @@ func (s *KeeperTestSuite) MustGetHostZone(chainId string) types.HostZone {
 	return hostZone
 }
 
-// Helper function to create an stride epoch tracker that dictates the timeout
-func (s *KeeperTestSuite) CreateStrideEpochForICATimeout(timeoutDuration time.Duration) {
+// Helper function to create an epoch tracker that dictates the timeout
+func (s *KeeperTestSuite) CreateEpochForICATimeout(epochType string, timeoutDuration time.Duration) {
 	epochEndTime := uint64(s.Ctx.BlockTime().Add(timeoutDuration).UnixNano())
 	epochTracker := types.EpochTracker{
-		EpochIdentifier:    epochtypes.STRIDE_EPOCH,
+		EpochIdentifier:    epochType,
 		NextEpochStartTime: epochEndTime,
 	}
 	s.App.StakeibcKeeper.SetEpochTracker(s.Ctx, epochTracker)
 }
 
+// Validates the query object stored after an ICQ submission, using some default testing
+// values (e.g. HostChainId, stakeibc module name, etc.), and returning the query
+// NOTE: This assumes there was only one submission and grabs the first query from the store
+func (s *KeeperTestSuite) ValidateQuerySubmission(
```

Comment:
Sorry thought I had responded to this!

I tested the callback data outside of this because I couldn't get it to work with go generics [[ref](https://github.com/Stride-Labs/stride/pull/1016#discussion_r1419714144)]